third_party/torch-mlir-dialects/BUILD

```diff
@@ -0,0 +1,151 @@
+load("@llvm-project//mlir:tblgen.bzl", "gentbl_cc_library", "td_library")
```

Comment:
Actually I am confused why these are in `third_party/...`. Ill reach out offline.